File: Python/crescente.py
print("Digite dois numeros: ")
num1 = int(input())
num2 = int(input())

while num1 != num2:
    if num1 < num2:
        print("CRESCENTE!")
    else :
        print("DECRESCENTE!")

    print("Digite outros dois numeros: ")
    num1 = int(input())
    num2 = int(input())

File: Linguagem C/Crescente.c
#include <stdio.h>

int main()
{
        int x, y;

        printf("Digite dois numero: \n");
        scanf("%i", &x);
        scanf("%i", &y);

        while (x!=y){
        if (x>y){
            printf("Decrescente\n");
        }
        else{
            printf("Crescente\n");
        }

        printf("Digite outros dois numero: \n");
        scanf("%i", &x);
        scanf("%i", &y);}
}
